Framework/API/test/ExperimentInfoTest.h

```diff
@@ -588,6 +588,27 @@ class ExperimentInfoTest : public CxxTest::TestSuite {
                                         instDir);
   }
 
+  void test_nexus_geometry_getInstrumentFilename() {
+
+    const auto originalInstDirectories =
```

Comment:
If you have moved the hdf4 files to now live in instrument, then this pushing and popping of data directories to instrument directories is not required.